packages/app-builder/src/components/ui/sidebar-item/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { cx } from '@linaria/core'
+import {
+  elBorderB,
+  elFlex,
+  elFlex1,
+  elFlexAlignCenter,
+  elFlexColumn,
+  elPx2,
+  ElSmallText,
+  elWFull,
+} from '@reapit/elements'
+import React, { FC } from 'react'
+
+import Arrow from '../../icons/arrow'
+import { bgWhite, cursorPointer, overflowAuto, uppercase } from '../styles'
+import { Chevron, HeaderDiv, IconContainer, SidebarItemDiv } from './styles'
+
+type SidebarItemProps = {
+  title: string
+  height?: string
+  icon: FC
+  visible?: boolean
+  onChange?: (bool: boolean) => void
+}
+
+const SidebarItem: React.FC<SidebarItemProps> = ({ visible, icon, title, children, height, onChange }) => {
+  return (
+    <SidebarItemDiv visible={visible} height={height} className={cx(elFlex, elFlexColumn)}>
+      <HeaderDiv
+        onClick={() => {
+          if (onChange) {
+            onChange(!visible)
+          }
+        }}
+        className={cx(cursorPointer, bgWhite, elBorderB, elFlex, elFlexAlignCenter, elPx2)}
+      >
+        <div className={cx(elFlex, elFlex1, elFlexAlignCenter)}>
+          <IconContainer>{icon}</IconContainer>
+          <ElSmallText className={cx(uppercase)} style={{ marginBottom: 0 }}>
```

Comment:
I would make this a suggestion but forgotten the name of the helper. There is a helper provided by @reapit/elements for margin bottom: 0. Might be mb0? 
```
          <ElSmallText className={cx(uppercase, mb0)}}>
```

Reply:
you're right that it's mb0, but the class from `ElSmallText` overrode `mb0` because of file ordering, so I had to put it here :/